pageserver/src/http/routes.rs

```diff
@@ -2639,5 +2706,13 @@ pub fn make_router(
         )
         .put("/v1/io_engine", |r| api_handler(r, put_io_engine_handler))
         .get("/v1/utilization", |r| api_handler(r, get_utilization))
+        .post(
+            "/v1/tenant/:tenant_shard_id/timeline/:timeline_id/ingest_aux_files",
+            |r| api_handler(r, ingest_aux_files),
+        )
+        .post(
+            "/v1/tenant/:tenant_shard_id/timeline/:timeline_id/list_aux_files",
+            |r| api_handler(r, list_aux_files),
```

Comment:
Just checking: is it intentional that you're not using the `testing_api_handler` here?

I definitely see the value this could have when troubleshooting production issues.